File: endpoint/templates/node/koa/typescript/endpoint.route.ts
import * as <%= name %>Controller from '../controller/<%= name %>-controller';

export default class <%= name %>Routes {
  static init(router) {
    let <%= name %> new <%= name %>Controller();

    router.get('/api/<%= nameLowerCase %>', <%= name %>.getAll);
    router.post('/api/<%= nameLowerCase %>', <%= name %>.createNew);
    router.del('/api/<%= nameLowerCase %>/:id', <%= name %>.removeById);
  }
}

File: endpoint/templates/go/echo/endpoint.route.go
package <%= nameLowerCase %>routes

import (
	"<%= repoHostUrl %>/<%= userNameSpace %>/<%= appName %>/server/api/<%= feature %>/controller"
	"github.com/labstack/echo"
)

func Init(e *echo.Echo) {
	e.GET("/api/<%= nameLowerCase %>", <%= nameLowerCase %>controller.GetAll)
	e.GET("/api/<%= nameLowerCase %>/:id", <%= nameLowerCase %>controller.GetById)
	e.POST("/api/<%= nameLowerCase %>", <%= nameLowerCase %>controller.New)
	e.POST("/api/<%= nameLowerCase %>/:id", <%= nameLowerCase %>controller.Update)
	e.DELETE("/api/<%= nameLowerCase %>/:id", <%= nameLowerCase %>controller.Remove)
}
